students/johnn/mailroom/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(
+    name='mailroom',
+    version='1.0.0',
+    author='johnn',
+    author_email='jnav@uw.edu',
+    packages=['mailroom', 'tests'],
+    scripts=['bin/mailroom'],
```

Comment:
so this should have put your mailroom script on the PATH when you install it -- does that not work?